src/explore-education-statistics-common/src/components/form/FormConditionalRadioGroup.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { Component, createRef, ReactNode } from 'react';
+
+interface Props {
+  children: ReactNode;
+}
+
+class FormConditionalRadioGroup extends Component<Props> {
```

Comment:
Don't think we need this as `FormRadioGroup` now has a `conditional` prop built into it already

Reply:
`FormRadioGroup` doesn't in the code I'm looking at. I just needed it to initialise the JavaScript.

Honestly, I'd be happy to completely remove the GovUK JavaScript, it's becoming a pain.

If the `FormRadioGroup` has the `conditional` property

Reply:
Oh sorry, I meant `FormRadio` has a conditional prop. `FormRadioGroup` accepts `FormRadioProps` for its `options` (with some exceptions). So usage would look like:

```javascript
<FormRadioGroup
  options={[
    { 
      value: 'something',
      label: 'something',
      conditional: <p>Some conditional text</p>
  ]}
/>
```